plugins/inputs/logfile/tail/tail.go

```diff
@@ -510,8 +510,13 @@ func (tail *Tail) sendLine(line string, offset int64) bool {
 		select {
 		case tail.Lines <- &Line{line, now, nil, offset}:
 		case <-tail.Dying():
-			tail.dropCnt += len(lines) - i
-			return true
+			if tail.Err() == errStopAtEOF {
+				// Try sending, even if it blocks.
+				tail.Lines <- &Line{line, now, nil, offset}
+			} else {
```

Comment:
Curious question (don't need to address in this PR): if there would be an initial stop request but some of the customers want to read until the EOF, would we want to stop the request or sending until it reaches EOF and sends those lines?

Reply:
`sendLine()` will keep getting called until `Tail` reaches EOF.
Which is the behavior _I think_ all users want and expect.